go.mod

```diff
@@ -5,6 +5,7 @@ require (
 	gotest.tools/v3 v3.3.0
 )
 
-require github.com/google/go-cmp v0.5.5 // indirect
+require  github.com/google/go-cmp v0.5.5 // indirect
+
 
 go 1.17
```

Comment:
go 1.17 is no longer supported, I suggest use the latest: https://endoflife.date/go